app/assets/stylesheets/modules/buttons.css.scss

```diff
@@ -161,10 +161,18 @@ button:disabled {
   }
 }
 
-.modal-button {
-  position: absolute;
-  right: 20px;
-  bottom: 20px;
+.whats-cwic {
```

Comment:
Style note (probably not worth fixing at the moment): This class name is heavily tied to the specific modal dialog. It'd be better to have it be something more generic/reusable in case we need other detailed help in the future.